### ISSUE TITLE
elcinema.com epg

### ISSUE THREAD

**Issue by yarafie** [iptv-org/epg#221]
I noticed that the elcinema.com epg is predominately in Arabic Language.

I modified the js slightly to get it mostly in English I've attached the js for your review and possibly you can create two version of the epg one in the default ARABIC language and the other in ENGLISH. 
[elcinema.com.en.config.js.txt](https://github.com/iptv-org/epg/files/7511098/elcinema.com.en.config.js.txt)

**Comment by yarafie** [comment]
That was quick cool updates.

How would I access the English version of the epg when it gets generated later?

**Comment by freearhey** [comment]
The link will appear in the [list](https://github.com/iptv-org/epg#usage) at the next update.

**Comment by yarafie** [comment]
Thanks @freearhey

**Comment by freearhey** [comment]
Arabic: `https://iptv-org.github.io/epg/guides/eg-ar/elcinema.com.epg.xml`
English: `https://iptv-org.github.io/epg/guides/eg-en/elcinema.com.epg.xml`

**Comment by yarafie** [comment]
Hi @freearhey just tested it and something looks strange regarding the timings of the programs.
The programs look fine themselves but the start times seem to be off.

After investigation have found that there is something wrong with their website determining the start time of the program based on your location I think it seems to be off.

I have visited their website using my phones 5G connection (I am GMT+3) and timings are correct. When I visit the same website using my local internet provider connection timing  seem to be off see attached pictures of one of the channels "MBC 2". The correct time for the show Jumanji is 09:00am which is correct when I generated the EPG from my Mobile 5G connection  but when I generate EPG using my ISP connection  it shows up as 01:00am and the follow on channels are also wrong and completely get screwed up when time changes from AM to PM so even by applying a timeshift in an IPTV player for EPG it won't be completely correct.

I am generating the EPG myself but also cross checked with the EPG  from links you provided they are consistent with the skewed time.

Just wanted to share in case someone else points out the wrong timings in EPG.


![Screenshot_20211111-101449_Chrome](https://user-images.githubusercontent.com/88841013/141255656-ecc32161-aa27-4620-8f46-447e478fdcd6.png)
![Screenshot_20211111-101416_Chrome](https://user-images.githubusercontent.com/88841013/141255660-6c2f1d0f-9aab-4738-812a-92aa9425cf49.png)
![Screenshot_20211111-091152_LazyIptv Deluxe](https://user-images.githubusercontent.com/88841013/141255664-20c048ad-f1d2-4134-845f-68301ff98335.png)
![Screenshot_20211111-091114_LazyIptv Deluxe](https://user-images.githubusercontent.com/88841013/141255668-7d0d2100-b438-4d74-abcf-09fd27a8ca03.png)
